<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pcbDialog</class>
 <widget class="QDialog" name="pcbDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Select One</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Create PCB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Delete PCB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Block </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Unblock</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Show PCB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Show All</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Show Ready</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Show Blocked</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Generate Random PCBs</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Execute </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Save</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Reload</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Load PCBs via File</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>System's Memory</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>301</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditRight">
        <property name="text">
         <string>HiOS! Command Line.. </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="commandButton">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>103</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="first_line_label">
        <property name="text">
         <string>PID #: </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="second_line_label">
      <property name="text">
       <string>Memory Need#: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>118</width>
     <height>92</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QSpinBox" name="firstLineBox"/>
      </item>
      <item>
       <widget class="QSpinBox" name="secondLineBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>28</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="processBt">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>131</y>
     <width>105</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QRadioButton" name="c0">
      <property name="text">
       <string>Regular</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="c1">
      <property name="text">
       <string>Round Robin</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="c2">
      <property name="text">
       <string>MLFQ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="labelLeftLineEdit">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>158</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditLeft"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonLeftLineEdit">
        <property name="text">
         <string>Load!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>161</y>
     <width>191</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="mlfq_label">
      <property name="text">
       <string># of Levels: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="mlfq_spin"/>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>comboBox</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
